source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -894,11 +894,13 @@ template<class NodeT>
 inline std::string ComponentInterface<NodeT>::validate_service_name(const std::string& service_name) {
   std::string parsed_service_name = utilities::parse_topic_name(service_name);
   if (parsed_service_name.empty()) {
-    throw exceptions::AddServiceException("Parsed service name is empty.");
+    throw exceptions::AddSignalException(
```

Comment:
You accidentally changes the exception type, it should still be `AddServiceException` here and a few lines below

Reply:
Ouf good catch thanks